Scrapper-Service/test/mongodb_test.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+from mdb import MongoDatabase
+from datamodels import Conference
+from datamodels import Metadata
+from datetime import datetime
+
+class MongoDbTestCase(unittest.TestCase):
+    def __init__(self, methodName):
+        super().__init__(methodName)
+    
+    def get_valid_data(self):
+        conference = Conference(  title="something", 
+                            url="anything", deadline=datetime.now(), 
+                            metadata=Metadata(__name__, datetime.now(), "something.com/something", "something.com","anythingProd" ))
+        
+        return MongoDatabase(database_name="Conference_Notify" , collection_name="conferences").put(conference)
+
```

Comment:
This is just a copy of the other test. test your logic inside here, write the test properly. test whether ur logic is working or not.